src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -447,16 +447,16 @@ void IdealLoopTree::reassociate_invariants(PhaseIdealLoop *phase) {
 // is applicable if we can make a loop-invariant test (usually a null-check)
 // execute before we enter the loop. When TRUE, the estimated node budget is
 // also requested.
-bool IdealLoopTree::policy_peeling(PhaseIdealLoop *phase) {
-  uint estimate = estimate_peeling(phase);
+bool IdealLoopTree::policy_peeling(PhaseIdealLoop* phase, bool scoped_value_only) {
```

Comment:
What exactly is the meaning of `scoped_value_only`?
That we only have a scoped value?

Or that we only should peel if we see a scoped value? If that: `peel_only_if_has_scoped_value`.